tests/testing_meshpy.py

```diff
@@ -541,6 +543,41 @@ def test_meshpy_geometry_sets(self):
         # Check the output.
         compare_test_result(self, mesh.get_string(header=False))
 
+    def test_unique_ordering_of_line_condition(self):
+        """
+        This test case ensures that the ordering of nodes defined within the line condition of the input file is unique.
```

Comment:
This test ensures that ordering of the nodes returned from the `get_all_nodes` function for line sets is unique.